src/main/java/frc/robot/Orchestrator.java

```diff
@@ -230,12 +239,12 @@ public Command pullBack() {
    * @return The command to move the robot and intake.
    */
   public Command driveWhileIntaking() {
-    return Commands.parallel(
-        intake.intake().until(() -> RobotState.getInstance().hasNote),
+    return Commands.race(
+        intakeBasic(),
         Commands.run(
                 () -> drive.runVelocity(new ChassisSpeeds(Units.inchesToMeters(10), 0.0, 0.0)),
                 drive)
-            .withTimeout(2));
+            .withTimeout(10));
```

Comment:
This means that the robot could drive at 10 inches per second for 10 seconds, so would drive about 100 inches (8.3 ft) if it doesn't detect that it picked up a note. It seems like that might be far enough that the robot would be on the other side of the field and get a penalty during auto.

Could we decrease the velocity and/or timeout to avoid that?